### PR TITLE
Remove duplicated operator<<

### LINKED ISSUE #281: operator<< for blaspp enums are duplicated.
Both util_blas.h and blaspp/enum.h contain the same output operator.

### PR DESCRIPTION
**Pull request by rasolca**
Closes #281.

### CODE REVIEW

**Comment by rasolca** [comment]
bors try

File: include/dlaf/blas/enum_output.h [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+namespace blas {
+inline std::ostream& operator<<(std::ostream& out, const blas::Diag& diag) {
+  switch (diag) {
+    case blas::Diag::Unit:
+      out << "Unit";
+      break;
+    case blas::Diag::NonUnit:
+      out << "NonUnit";
+  }
```
Comment (Sely85) [inline include/dlaf/blas/enum_output.h:24]: should we add a `break;` in this `case` as well?

```suggestion
  break;
  }
```

**Review by Sely85 — CHANGES_REQUESTED** [review]

**Review by teonnik — APPROVED** [review]

**Review by Sely85 — APPROVED** [review]
Looks good!

**Comment by albestro** [comment]
LGTM too!

Just one question: I quickly run grep over the repo and there are few locations where we are using `blaspp` provided one. Wouldn't it be a nice chance to migrate them?

```
include/dlaf/auxiliary/norm/mc.tpp:      DLAF_ASSERT(false, "not yet implemented", lapack::norm2str(norm_type));
include/dlaf/auxiliary/norm/mc.tpp:          DLAF_ASSERT(false, "not yet implemented", lapack::norm2str(norm_type), blas::uplo2str(uplo));
test/unit/auxiliary/mc/test_norm.cpp:  SCOPED_TRACE(::testing::Message() << "norm=" << lapack::norm2str(norm_type)
test/unit/auxiliary/mc/test_norm.cpp:                                    << " uplo=" << blas::uplo2str(uplo) << " changed element=" << index
test/unit/test_lapack_tile/test_lange.h:  SCOPED_TRACE(::testing::Message() << "LANGE: " << lapack::norm2str(norm) << ", " << a.size()
test/unit/test_lapack_tile/test_lange.h:      FAIL() << "norm " << lapack::norm2str(norm) << " is not supported by lange";
test/unit/test_lapack_tile/test_lantr.h:  SCOPED_TRACE(::testing::Message() << "LANTR: " << lapack::norm2str(norm) << ", " << a.size()
test/unit/test_lapack_tile/test_lantr.h:                                    << ", ld = " << a.ld() << " uplo = " << blas::uplo2str(uplo)
test/unit/test_lapack_tile/test_lantr.h:                                    << " diag = " << blas::diag2str(diag));
test/unit/test_lapack_tile/test_lantr.h:      FAIL() << "norm " << lapack::norm2str(norm) << " is not supported by lantr";
```

**Review by albestro — APPROVED** [review]
LGTM too!

Just one question: I quickly run grep over the repo and there are few locations where we are using `blaspp` provided one. Wouldn't it be a nice chance to migrate them?

```
include/dlaf/auxiliary/norm/mc.tpp:      DLAF_ASSERT(false, "not yet implemented", lapack::norm2str(norm_type));
include/dlaf/auxiliary/norm/mc.tpp:          DLAF_ASSERT(false, "not yet implemented", lapack::norm2str(norm_type), blas::uplo2str(uplo));
test/unit/auxiliary/mc/test_norm.cpp:  SCOPED_TRACE(::testing::Message() << "norm=" << lapack::norm2str(norm_type)
test/unit/auxiliary/mc/test_norm.cpp:                                    << " uplo=" << blas::uplo2str(uplo) << " changed element=" << index
test/unit/test_lapack_tile/test_lange.h:  SCOPED_TRACE(::testing::Message() << "LANGE: " << lapack::norm2str(norm) << ", " << a.size()
test/unit/test_lapack_tile/test_lange.h:      FAIL() << "norm " << lapack::norm2str(norm) << " is not supported by lange";
test/unit/test_lapack_tile/test_lantr.h:  SCOPED_TRACE(::testing::Message() << "LANTR: " << lapack::norm2str(norm) << ", " << a.size()
test/unit/test_lapack_tile/test_lantr.h:                                    << ", ld = " << a.ld() << " uplo = " << blas::uplo2str(uplo)
test/unit/test_lapack_tile/test_lantr.h:                                    << " diag = " << blas::diag2str(diag));
test/unit/test_lapack_tile/test_lantr.h:      FAIL() << "norm " << lapack::norm2str(norm) << " is not supported by lantr";
```

**Comment by rasolca** [comment]
bors try

**Comment by albestro** [comment]
bors try

In local I had an old version of lapackpp, now the header has been moved, it should be ok now

**Comment by rasolca** [comment]
bors try
Something external failed.

**Comment by rasolca** [comment]
bors try

**Review by albestro — APPROVED** [review]

**Comment by albestro** [comment]
I was viewing changes after the merge, but I didn't touch anything, but it seems there was a pending review (not sure, no messages, no comments, just files marked as view).

Anyway, it seems the maintenance is still ongoing, we may have to wait.

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Pull request merged**